### DISCUSSION TITLE
Questions on Parent Child Inscriptions

### DISCUSSION THREAD

**Discussion by skyrover7** [ordinals/ord#2880] [Q&A] [answered]
I know the ordinals docs does provide some brief examples of the bitcoin script required for creating a parent child inscription and it does seem straightforward in ord wallet. However if creating the transaction and envelope manually I was wondering how does one `Spend the parent P in one of the inputs of T.`?

**Comment by raphjaph** [comment, marked as answer]
You have to manually construct the transaction by selecting as one of the inputs the one that contains the parent inscriptions. If you're interested how we do it have a look at the file `src/subcommand/wallet/inscribe/batch.rs` lines 209 and down.

**Comment by skyrover7** [comment]
thanks @raphjaph that's what i figured! thanks for the code ref. still fumbling about in bitcoin script trying to get it right. 

can i use an inscription that i created from ord client when i no longer have access to that client? I do have access to the inscription.

also can a parent have multiple chlidren? does anything special need to be done with the parent to support that?

**Reply by raphjaph** [reply]
You need to own the parent inscription. You can create multiple children in one transaction, you just have to pass in the parent in one of the inputs and specify the parent in all children. Here's an example parent with children: https://ordinals.com/inscription/17541f6adf6eb160d52bc6eb0a3546c7c1d2adfe607b1a3cddc72cc0619526adi0

**Reply by skyrover7** [reply]
got it, but can those children be created separately rather than in one transaction?

**Reply by raphjaph** [reply]
yes you can do it sequentially as well. There's no limit to how many children you can create under one parent. The command with ord would be `ord wallet inscribe --parent <PARENT_ID> --file <CHILD_INSCRIPTION>`

**Reply by rohitkeshav27** [reply]
I was trying to add multiple parent to a child inscription .
So , according the the "Ordinal Theory Handbook" , 
![image](https://github.com/ordinals/ord/assets/43274184/136661c1-9b8a-405b-b771-7c2bc6878db8)

I follow the first 2 points .According to the second bullet point I spend both parent inscription as first 2 inputs , and the creation of the child inscription is the 3rd input.

But my doubt is in the 3rd bullet point , Which serialised binary inscription ID should I include in the _tag 3_ the first parent's or the seconds , if i have to add both how do i do it?

**Reply by rohitkeshav27** [reply]
@skyrover7  Could you please help me with the code part of it ?
Or connect me with whom I can get my doubts clarified?

**Reply by raphjaph** [reply]
you include two tag 3 fields with the respective parent ids

**Reply by rohitkeshav27** [reply]
Yeah thanks alot.

I have noticed sometimes 2 parents are shown and sometimes only one parent on Ordinals.com ,

Why is that the case ?

**Comment by rohitkeshav27** [comment]
Thanks alot !

**Comment by rohitkeshav27** [comment]
But I have noticed, sometimes both the parents are shown and sometimes only
one on Ordinals.com?
Why is that the case?

**Reply by raphjaph** [reply]
It is allowed to specify any amount of parents.

**Reply by rohitkeshav27** [reply]
I understand it can support any number of parents, but why does it not show sometimes in the Ordinals.com?
